v22.1/aws-dms.md

```diff
@@ -25,6 +25,18 @@ Ensure the following items are completed prior to starting this tutorial:
 - Configure a [replication instance](https://docs.aws.amazon.com/dms/latest/userguide/CHAP_ReplicationInstance.html) in AWS.
 - Ensure you have a secure, publicly available CockroachDB cluster running v22.1 GA or later.
 
+The following databases are supported for use as source databases in AWS DMS:
+
+- Oracle 
+- Microsoft SQL Server
+- Microsoft Azure SQL
+- MySQL-compatible databases (including MariaDB)
+- PostgreSQL-compatible databases (including Amazon Aurora)
+- SAP ASE 
+- MongoDB/Amazon DocumentDB (with MongoDB compatibility) 
+- Amazon S3
```

Comment:
S3 is an object store, not a database. Does that matter?

Reply:
i let this and the mongo one go because i was thinking "this is what is supported as source databases in AWS DMS", but i guess i'd say it'd be safe to delete the SAP/Mongo/S3 lines.